static/app/components/replays/replayContext.tsx

```diff
@@ -33,6 +34,7 @@ const ReplayPlayerContext = React.createContext<ReplayPlayerContextProps>({
   dimensions: {height: 0, width: 0},
   duration: undefined,
   events: [],
+  fastForwardSpeed: 0,
```

Comment:
Is this different from `speed`? Can we add a comment noting the difference between the two?

Reply:
good idea, I added docs to all the values that the context returns.

This one specifically is only set if the video is fast-forwarding right now, which the other speed is the requested playback.